pkg/ccl/changefeedccl/kvfeed/scanner.go

```diff
@@ -191,6 +191,7 @@ func (p *scanRequestScanner) exportSpan(
 		r := kvpb.NewScan(remaining.Key, remaining.EndKey).(*kvpb.ScanRequest)
 		r.ScanFormat = kvpb.BATCH_RESPONSE
 		b.Header.TargetBytes = targetBytesPerScan
+		b.Header.ConnectionClass = kvpb.ConnectionClass_BULK_DATA
```

Comment:
Should we hook this in to the [env var](https://github.com/cockroachdb/cockroach/blob/9b10f083f8889cdc1bc549d9428f06fd784ba081/pkg/kv/kvclient/kvcoord/dist_sender_rangefeed.go#L58-L66) that overrides to the default class?

We could also add a separate env var for this here.

More generally, we already have 2 env vars for overriding connection classes. Since this is a third candidate for such a setting, I wonder if we should generalize all these cases into a single env var that's a key-value list of all the overrides. Something like:

`COCKROACH_USE_DEFAULT_CONNECTION_CLASS=raft,rangefeed` - resets to default class for rangefeed and raft traffic.
`COCKROACH_USE_DEFAULT_CONNECTION_CLASS=*` - resets to default class for all traffic types in question.

Reply:
Wrote a quick and dirty prototype: #117882.

Reply:
I don't think we need to overdo it, we mostly need this as an escape hatch in case something breaks with the separate connection classes (not sure what that would be -- maybe some environments have limits on concurrent TCP connections).

I think a common `COCKROACH_USE_DEFAULT_CONNECTION_CLASS=true|false` would go a long way.

Reply:
Added another commit with this env var that reset every. It completely resets the behaviour to the previous state when we only had "default" and "system".